contracts/mocks/docs/account/MyAccountCustom.sol

```diff
@@ -0,0 +1,44 @@
+// contracts/MyAccountCustom.sol
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {EIP712} from "@openzeppelin/contracts/utils/cryptography/EIP712.sol";
+import {ERC4337Utils, PackedUserOperation} from "@openzeppelin/contracts/account/utils/draft-ERC4337Utils.sol";
+import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import {AccountBase} from "../../../account/draft-AccountBase.sol";
+import {ERC7739Signer} from "../../../utils/cryptography/draft-ERC7739Signer.sol";
+
+contract MyAccountCustom is ERC7739Signer, AccountBase, Initializable {
```

Comment:
This mixes vanilla contracts (EIP712) with an upgradeable design (the initializer).

I'm wondering what will happen when we get an upgradeable version of the community repo. Like `AccountBaseUpgradeable`?